Library/Homebrew/cask/lib/hbc/cask.rb

```diff
@@ -85,6 +85,15 @@ def installed?
       !versions.empty?
     end
 
+    def latest_installed_version
+      versions.last
+    end
+
+    def outdated?(greedy: false)
+      (greedy || !auto_updates && !version == :latest) &&
+      !version.nil? && (version.casecmp latest_installed_version) == 1
```

Comment:
> We need the casecmp for a later than check rather than an equality check.

How is a case comparison helping with deciding whether a version is later than another given version? I think it's safe to assume that any other version other than the installed one is “later“, since I have never seen a Cask get downgraded.

So just make this `version != latest_version_installed`.